.gitignore

```diff
@@ -74,3 +74,4 @@ docs/cache
 
 !style-guide/vendor
 !style-guide/index.php
+
```

Comment:
Sometimes in your pull requests, there are new lines appearing at the end of files like this one. In your PHPStorm, in Preferences > Editor > General, is the `Ensure every saved file ends with a line break` option checked?